edgelet/build/linux/package.sh

```diff
@@ -114,6 +114,7 @@ case "$PACKAGE_OS.$PACKAGE_ARCH" in
                 curl git make rpm-build \
                 gcc gcc-c++ \
                 libcurl-devel libuuid-devel openssl-devel &&
+            git config --global --add safe.directory \'*\'
```

Comment:
This is intentional to stabilize checkin. Didn't want to spin on multiple PRs for simple build change. Same thing was checked into 1.4.